### ISSUE TITLE
`this.load` is not a function (with @rollup/plugin-commonjs)

### ISSUE THREAD

**Issue by cdauth** [PepsRyuu/nollup#230]
I use `@rollup/plugin-commonjs`. When I run `nollup`, I get the following error:
```
[Nollup] Listening on http://localhost:8080
/home/cdauth/Documents/workspace/programming/web/osm/facilmap/node_modules/@rollup/plugin-commonjs/dist/cjs/index.js:597
      const moduleInfo = await this.load(resolved);
                                    ^

TypeError: this.load is not a function
    at Object.resolveId (/home/cdauth/Documents/workspace/programming/web/osm/facilmap/node_modules/@rollup/plugin-commonjs/dist/cjs/index.js:597:37)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async PluginErrorHandler.wrapAsync (/home/cdauth/Documents/workspace/programming/web/osm/facilmap/node_modules/nollup/lib/impl/PluginErrorHandler.js:98:22)
    at async _callAsyncHook (/home/cdauth/Documents/workspace/programming/web/osm/facilmap/node_modules/nollup/lib/impl/PluginLifecycle.js:25:21)
    at async callAsyncFirstHook (/home/cdauth/Documents/workspace/programming/web/osm/facilmap/node_modules/nollup/lib/impl/PluginLifecycle.js:54:18)
    at async Object.resolveIdImpl (/home/cdauth/Documents/workspace/programming/web/osm/facilmap/node_modules/nollup/lib/impl/PluginLifecycle.js:205:18)
    at async Object.resolve (/home/cdauth/Documents/workspace/programming/web/osm/facilmap/node_modules/nollup/lib/impl/PluginContext.js:122:28)
    at async Object.resolveId (/home/cdauth/Documents/workspace/programming/web/osm/facilmap/node_modules/@rollup/plugin-node-resolve/dist/cjs/index.js:1182:34)
    at async PluginErrorHandler.wrapAsync (/home/cdauth/Documents/workspace/programming/web/osm/facilmap/node_modules/nollup/lib/impl/PluginErrorHandler.js:98:22)
    at async _callAsyncHook (/home/cdauth/Documents/workspace/programming/web/osm/facilmap/node_modules/nollup/lib/impl/PluginLifecycle.js:25:21) {
  __isNollupError: true
}

Node.js v18.1.0
```

**Comment by PepsRyuu** [comment]
Looks like a new plugin context API method. Will investigate when I can! :)

**Comment by PepsRyuu** [comment]
Looking into this, it seems like this requires some pretty big changes to the architecture, so it's going to take a bit longer to figure this out. In the meantime, just downgrade to a previous version of the CommonJS plugin, or alternatively you can use ```rollup-plugin-commonjs-alternate```.

**Comment by PepsRyuu** [comment]
https://github.com/PepsRyuu/nollup/pull/235/files Initial version complete. Need to do some final testing and can release.

**Comment by PepsRyuu** [comment]
Fixed in ```0.20.0```. Note that ```--live-bindings``` must be enabled to use ```@rollup/plugin-commonjs``` and there are known issues with HMR when using live bindings. It is still preferable to use ```rollup-plugin-commonjs-alternate``` which should support vast majority of CommonJS libraries.
